ecs-init/docker/docker.go

```diff
@@ -440,6 +459,22 @@ func (c *client) getHostConfig(envVarsFromFiles map[string]string) *godocker.Hos
 	return createHostConfig(binds)
 }
 
+// getCredentialsFetcherSocketBind returns the corresponding bind for credentials fetcher socket.
+func getCredentialsFetcherSocketBind() (string, bool) {
+	CredentialsFetcherUnixSocketHostPath := config.CredentialsFetcherUnixSocket()
+	CredentialsFetcherUnixSocketAgentPath := CredentialsFetcherUnixSocketHostPath
```

Comment:
nit: do we need a new var `CredentialsFetcherUnixSocketAgentPath`?

could we just return `CredentialsFetcherUnixSocketHostPath + ":" + CredentialsFetcherUnixSocketHostPath`

Reply:
just to make it visible about the bind mount

Reply:
we're doing this today as well.

example: `binds = append(binds, gpu.GPUInfoDirPath+":"+gpu.GPUInfoDirPath)` above around L444

Reply:
refactored variables accordingly